App/tests/test_app.py

```diff
@@ -69,32 +69,28 @@ def test_author_toJSON(self):
 
 class PublicationUnitTests(unittest.TestCase):
     def test_new_publication(self):
-        authors = []
         coauthors = []
         author = Author("Bob Moog", "05/08/2001", "BSc. Computer Science")
-        authors.append(author)
         coauthor = Author("Bob Dule", "06/09/2002", "BSc. Computer Engineering")
         coauthors.append(coauthor)
-        publication = Publication("Intro to Computer Science", authors, coauthors)
+        publication = Publication("Intro to Computer Science", author.id, coauthors)
         assert (
             publication.title=="Intro to Computer Science" 
-            and publication.authors==authors 
+            and publication.authorId==author.id
             and publication.coauthors==coauthors
         )
 
     def test_publication_toJSON(self):
-        authors = []
         coauthors = []
         author = Author("Bob Moog", "05/08/2001", "BSc. Computer Science")
-        authors.append(author)
         coauthor = Author("Bob Dule", "06/09/2002", "BSc. Computer Engineering")
         coauthors.append(coauthor)
-        publication = Publication("Intro to Computer Science", authors, coauthors)
+        publication = Publication("Intro to Computer Science", author.id, coauthors)
         publication_json = publication.toJSON()
         self.assertDictEqual(publication_json, {
             "id": None,
             "title": "Intro to Computer Science",
-            "authors": [author.toJSON() for author in authors],
+            "author": None,
```

Comment:
Would this return the author.id as null? I think sqlalchemy auto-generates a primary key here.

Reply:
When I did the tests from the last project the id for unit tests was returned as null.

Reply:
Huh alright, I'll have to read up on that.